apps/fares/lib/format.ex

```diff
@@ -83,10 +83,10 @@ defmodule Fares.Format do
   def name(:local_bus), do: "Local Bus"
   def name(:inner_express_bus), do: "Inner Express Bus"
   def name(:outer_express_bus), do: "Outer Express Bus"
-  def name(:ferry_inner_harbor), do: "Inner Harbor Ferry"
+  def name(:ferry_inner_harbor), do: "Charlestown Ferry"
```

Comment:
Would it make sense to rename the atoms as well not only the strings?

Reply:
I think the atoms should resemble the GTFS system route names more than human label. That being said, not sure what those are off the top of my head but I assumed the current atoms lined up with GTFS.

Reply:
Ah ok disregard then